docs/topics/mta-web-assessment-module.adoc

```diff
@@ -5,78 +5,10 @@
 
 :_content-type: REFERENCE
 [id="mta-web-assessment-module_{context}"]
-= Assessment Module
+= Assessing an application
 
 The assessment process for applications that are to be migrated in {ProductFullName} has been separated from other features.
 
-In the {ProductShortName} user interface, the Assessment Module appears in a separate menu item. Selecting the item shows the currently saved applications.
+In the {ProductShortName} user interface, you can assess an application and display the currently saved applications by using the Assessment module.
```

Comment:
Do you mean 'currently saved assessments' instead of 'currently saved application' ?

Reply:
Not sure here, was just fixing the original text :-D S, should it be "assessments" instead?

Reply:
Good! Has already been fixed :-)